client/.storybook/main.js

```diff
@@ -1,3 +1,3 @@
 module.exports = {
-    addons: ['@storybook/addon-actions/register']
+    stories: ['../stories/*.stories.jsx']
```

Comment:
Thanks for getting Storybook working, this should definitely come in handy in the future!

Reply:
Could you document the command to run storybook in the readme? Just the `yarn storybook` so people know it's there.

Reply:
All set!